src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -104,6 +107,7 @@ public CryptoProviderFactory(CryptoProviderFactory other)
             if (other == null)
                 throw LogHelper.LogArgumentNullException(nameof(other));
 
+            CryptoProviderCache = new InMemoryCryptoProviderCache();
```

Comment:
should we create if null?

Reply:
This is consistent with the behavior that was prior. Wanted to keep the behavior consistent